Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -0,0 +1,96 @@
+import os
+import vtk, qt, ctk, slicer
+import logging
+
+#########################################################
+#
+#
+comment = """
+
+  DICOMUtils is a collection of static-like DICOM
+  utility functions facilitating convenient scripted
+  use of the DICOM database
+
+"""
+#
+######################################################### 
+
+#------------------------------------------------------------------------------
+# Get patient UID by patient name for easy loading of a patient
+def patientUIDByPatientName(name):
```

Comment:
This method name is not very clear for me. getPatientUIDByPatientName or patientUIDFromPatientName would be more clear.